tests/baselines/reference/config/convertCompilerOptionsFromJson/Allow trailing comments with jsonSourceFile api.js

```diff
@@ -12,7 +12,7 @@ Fs::
 configFileName:: tsconfig.json
 CompilerOptions::
 {
-  "configFilePath": "tsconfig.json"
+  "configFilePath": "/apath/tsconfig.json"
```

Comment:
I've been trying to find the code that changed to make these baselines change; do you know why this changed like this?

Reply:
Its from https://github.com/microsoft/TypeScript/pull/58042/commits/7b1a84825a836557619e100565d51d71d98b1494